File: 백준/Bronze/2083. 럭비 클럽/럭비 클럽.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while (true) {
            String input = sc.nextLine();
            
            if (input.equals("# 0 0")) {
                break;
            }

            String[] parts = input.split(" ");
            String name = parts[0];
            int age = Integer.parseInt(parts[1]);
            int weight = Integer.parseInt(parts[2]);

            if (age > 17 || weight >= 80) {
                System.out.println(name + " Senior");
            } else {
                System.out.println(name + " Junior");
            }
        }

        sc.close();
    }
}

File: 백준/Bronze/2083. 럭비 클럽/럭비 클럽.py
while 1:
    query = input()
    if query == '# 0 0':
        break
    name, age, weight = query.split()
    
    if int(age) > 17 or int(weight) >= 80:
        print(name + ' Senior')
    else:
        print(name + ' Junior')